### DISCUSSION TITLE
show section by default

### DISCUSSION THREAD

**Discussion by toppits** [openstyles/stylus#1293] [Q&A] [answered]
It's collapsed by default for me, but imo, if there are sections, they'll be accessed frequently.

Any way to make it open by default?

**Comment by tophf** [comment, marked as answer]
This is a bug. It'll be fixed in the next release. Meanwhile you can either use the older stable Stylus or load the newer fixed Stylus from the `master` branch as an unpacked extension, see https://github.com/openstyles/stylus/wiki/Install-Stylus-from-GitHub
